velox/dwio/common/MetadataFilter.h

```diff
@@ -31,7 +32,10 @@ class MetadataFilter {
 
   /// Construct from a ScanSpec and an expression.  The leaf filter nodes
   /// generated will be added to the corresponding position in ScanSpec.
-  explicit MetadataFilter(ScanSpec&, const core::ITypedExpr&);
+  explicit MetadataFilter(
+      ScanSpec&,
+      const core::ITypedExpr&,
+      core::ExpressionEvaluator*);
```

Comment:
nit: could add variable name for ctor input args for readability? Thanks!

Reply:
The variable names here won't be more descriptive than the type names